initials.c

```diff
@@ -0,0 +1,10 @@
+//Olivier Gabison - Wrote a simple piece of code depicting my initials.
+#include <stdio.h>
+
+int main() {
+printf("*******   *******");
```

Comment:
you need an \n at the end of each line so that it breaks to the next line. Output was all on one line instead. Aside from that, you also forgot to include a return 0; at the end.